tests/components/blink/test_init.py

```diff
@@ -0,0 +1,208 @@
+"""Test the Blink init."""
```

Comment:
I would suggest to do the tests in another PR as that doesn't have to do anything with the coordinator

Reply:
Done...